toto_client/toto_client.py

```diff
@@ -199,8 +199,24 @@ def search_term(self, search_term):
                   id
                   fileName
                   dataType
-                  pageNumber
-                  pageIndexes
+                  regs: datas(tagName: "regs") {
+                    id
+                        dataType
+                        pageNumber
+                        text
+                    }
+                    effects: datas(tagName: "effects") {
+                        id
+                        dataType
+                        pageNumber
+                        text
+                    }
+                    sub_evidence: datas(tagName: "sub_evidence") {
+                        id
+                        dataType
+                        pageNumber
+                        text
+                    }
```

Comment:
Great work, this is exactly the query we want!

It will be ideal if we can generalise like Linus suggested yesterday - take a look at how the `get_data` method works and update this to work in the same way 
i.e. here the tags are hard coded, we want instead to input a list of tags and generate the query from those tags. 

Happy to have a chat if you need a hand :)